components/style/properties/longhand/inherited_text.mako.rs

```diff
@@ -750,6 +751,17 @@ ${helpers.single_keyword("text-align-last",
         pub color: Option<specified::CSSColor>,
     }
 
+    impl From<Shadow> for SpecifiedTextShadow {
+        fn from(shadow: Shadow) -> SpecifiedTextShadow {
```

Comment:
Given it's pretty much the same type, and they share parsing, the only thing that differs is the `to_computed_value` impl, right?